core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -226,6 +230,14 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
 
   def produceMessageConversionsRate = metricTypeMap.get(BrokerTopicStats.ProduceMessageConversionsPerSec).meter()
 
+  def noKeyCompactedTopicRecordsPerSec = metricTypeMap.get(BrokerTopicStats.NoKeyCompactedTopicRecordsPerSec).meter()
```

Comment:
Should the return type be Option[meter] to be uniform with other `allTopicStats` only metrics?